curriculum/challenges/english/25-front-end-development/quiz-semantic-html/66ed903cf45ce3ece4053ebe.md

```diff
@@ -897,7 +897,7 @@ ordinal numbers
 
 #### --answer--
 
-foot notes
+Chemical formula
```

Comment:
you should make this all lowercase, or give proper capitalization to the distractors